src/CMakeLists.txt

```diff
@@ -209,12 +209,20 @@ else()
     add_library (clickhouse_new_delete STATIC Common/new_delete.cpp)
     target_link_libraries (clickhouse_new_delete PRIVATE clickhouse_common_io)
     if (TARGET ch_contrib::jemalloc)
+        message("src/cmake new_delete jemalloc")
```

Comment:
By the way, you have not disabled jemalloc, hence mimalloc is not used in the build